packages/gatsby-plugin-netlify-cms/src/gatsby-node.js

```diff
@@ -36,6 +36,7 @@ exports.onCreateWebpackConfig = (
   { store, stage, getConfig, plugins, pathPrefix },
   {
     modulePath,
+    modulePaths = modulePath,
```

Comment:
I'm not sure if it's worth to change the field name here (probably could be more confusing for people comparing their current config to documentation when they see that they use `modulePath` and not `modulePaths` but it still works? I would think just adding type and potentially adding note that it can accept single path or array of paths would be good

Reply:
Cool! I've kept the name instead of renaming